doc/src/CONTRIBUTING.md

```diff
@@ -1,253 +1,65 @@
 
-```
-git clone https://gitlab.com/lfortran/lfortran.git
-cd lfortran
-git remote add REMOTE_NAME git@gitlab.com:YOUR_GITLAB_ID/lfortran.git
-```
-:fontawesome-solid-edit: `REMOTE_NAME` is the name of your remote repository and could be any name you like, for example your first name.
+- Create a new branch
+    ```bash
+    git checkout -b [branch-name]
+    ```
+    Replace `[branch-name]` with a suitable name relevant to the changes you desire to make. For example, if you plan to fix an issue, you can name your branch `fixes-issue25`.
 
-:fontawesome-solid-edit: `YOUR_GITLAB_ID` is your user ID on GitLab and should be part of your account path.
 
-You can use `git remote -v` to check if the new remote is set up correctly.
+- Make changes in the relevant file(s).
+  Test the changes you make. Try building the project. Run some tests to verify your code does not break anything.
 
-### Send a New Merge Request
+  Check the [installation-docs](./installation.md) for more details on building and testing.
 
-Step 1. Create a new branch
-```
-git checkout -b fix1
-```
+- Once you are assured, go ahead and push a commit with the changed files.
+    ```bash
+    git add FILE1 (FILE2 ...)
+    git commit -m "YOUR_COMMIT_MESSAGE"
+    ```
+    Replace `FILE1 (FILE2 ...)` with the file names (with file extension) of changed files.
 
-Step 2. Make changes in relevant file(s)
+    Write a short but descriptive commit message. [Here](https://chris.beams.io/posts/git-commit/) are some great tips on writing good commit messages.
 
-Step 3. Commit the changes:
+- Open a Pull Request (PR)
+    - Go to your project fork on GitHub.
+    - Select the branch which contains your commit.
+      ![image](https://github.com/kmr-srbh/lpython/assets/151380951/fb595307-9610-44f3-89d0-0079a90fcf9e)
+    - Above the list of files, select the green button which says <kbd>Compare & pull request</kbd>. If you do not see this option, select the <kbd>Contribute</kbd> option.
+      ![Screenshot from 2024-02-17 11-52-41](https://github.com/kmr-srbh/lpython/assets/151380951/08b3dc25-f5d1-4624-9887-2ede565230bc)
+    - Type a descriptive title. Provide further details in the Description section below. If your PR fixes an issue, link it with the issue by appending `fixes #[issue-number]` to your PR title.
+      ![image](https://github.com/kmr-srbh/lpython/assets/151380951/3f2245df-42f4-44e5-9c20-d6d7789ac894)
+    - Once you are done, select the <kbd>Create Pull Request</kbd> button to open a new PR.
+      
+  Congratulations! You just made your first contribution!
+  
+### What next?
+Sit back and relax or go on a bug hunt.
 
-```
-git add FILE1 (FILE2 ...)
-git commit -m "YOUR_COMMIT_MESSAGE"
-```
+We too have day jobs and reviewing a Pull Request (PR) may take some time. We request you to patiently wait for any feedback or action on your PR. If you are asked to make changes, push commits to the same branch to automatically include them in the open PR.
```

Comment:
Not sure if we need this line.

Reply:
I understand your point @Shaikh-Ubaid . I too was hesitant adding this paragraph seeing the sheer amount of students contributing to the project. 

Please tell me the changes you recommend. Shall we remove it completely, or make adjustments? Reviewing a PR does take time, right?

Reply:
@certik could you share what should be done here?